### PR TITLE
reworking service controller release roadmap

### PR DESCRIPTION
**Pull request by jaypipes**
Customers and users are looking for a clear visual roadmap that shows
roughly when they should expect certain AWS services to get an ACK
controller into developer preview, beta and GA release statuses.

We will be re-shaping the projects/1 GH project to contain fine-grained
tasks and this PR reworks the services.md and releases.md documents to
turn the services.md doc into more of a clear tabular roadmap of service
controllers, with links to milestones (that can be date-targeted) for
individual service controllers.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/contents/services.md [hunk 1]
```diff
@@ -1,26 +1,149 @@
...
+| Amazon [Lambda][lambda] | `PLANNED` |
+| Amazon [MQ][mq] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/12)
+| Amazon [MSK][msk] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/13)
+| Amazon [RDS][rds] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/8)
+| Amazon [Sagemaker][sagemaker] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/11)
+| Amazon [SNS][sns] | `DEVELOPER PREVIEW` |
+| Amazon [SQS][sqs] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/6)
+| Amazon [StepFunctions][sfn] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/10)
+| Amazon [S3][s3] | `DEVELOPER PREVIEW` |
+
```
Comment (dbyron0) [inline docs/contents/services.md:28]: I'm curious about plans for IAM.  There's https://github.com/aws/aws-controllers-k8s/issues/222, but does it make sense to add something here too?

Comment (jaypipes) [reply docs/contents/services.md:28]: @dbyron0 yes :) As soon as this PR is merged, feel like pushing a PR that adds that?

File: .github/ISSUE_TEMPLATE/propose_new_controller.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## New ACK Service Controller
+
+Support for [service name]
+
+### List of API resources
+
+List the API resources in order of importance to you:
+
+* [resource 1]
```
Comment (tabern) [inline .github/ISSUE_TEMPLATE/propose_new_controller.md:18]: should this be numbered?
1.
2. etc

**Review by tabern — COMMENTED** [review]
really like this. Should we update the template to include numbers instead of bullets

File: docs/contents/services.md (hunk 1 above)
Comment (dbyron0) [reply docs/contents/services.md:28]: Sure, if `PLANNED` is the right value for the `Current Status` column, with `Next Milestone` blank.

**Review by tabern — APPROVED** [review]
/lgtm

File: docs/contents/services.md (hunk 1 above)
Comment (jaypipes) [reply docs/contents/services.md:28]: It would be PROPOSED for IAM, since we don't yet have set plans for it :)

Comment (dbyron0) [reply docs/contents/services.md:28]: Like [this](https://github.com/aws/aws-controllers-k8s/pull/495)?  I also saw https://github.com/aws/aws-controllers-k8s/issues/484 fly by.

**Pull request merged**